r/rubyzip/rubyzip_ubi_8.4.sh

```diff
@@ -0,0 +1,205 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : rubyzip
+# Version       : v2.2.0
+# Source repo   : https://github.com/rubyzip/rubyzip.git
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Siddhesh Ghadi <Siddhesh.Ghadi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e 
+
+PACKAGE_NAME=rubyzip
+PACKAGE_VERSION=${1:-v2.2.0}
+PACKAGE_URL=https://github.com/rubyzip/rubyzip.git
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq curl make gcc-c++ procps gnupg2 ruby libcurl-devel libffi-devel ruby-devel redhat-rpm-config sqlite sqlite-devel java-1.8.0-openjdk-devel rubygem-rake
+
+yum install -y --allowerasing gcc gcc-c++ yum-utils make automake autoconf libtool gdb* binutils rpm-build gettext wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
```

Comment:
@Siddhesh-Ghadi I can see that the centos repo  enablement code is coming from the Ruby template, but do we need the dependencies being installed from there specifically for this package? Basically, I would like to avoid enabling centos, if possible as not everyone might want to do that? also please check if the packages needed from there (if any in particular) are available from EPEL or otherwise on RHEL/

Reply:
Removed usage of centos repos.